source/common/config/cds_json.cc

```diff
@@ -33,6 +33,9 @@ void CdsJson::translateHealthCheck(const Json::Object& json_health_check,
   const std::string hc_type = json_health_check.getString("type");
   if (hc_type == "http") {
     health_check.mutable_http_health_check()->set_path(json_health_check.getString("path"));
+    if (json_health_check.hasObject("host")) {
```

Comment:
Do we need v1 support? If so we need to add v1 docs. Otherwise I would just drop it.

Reply:
I have removed the v1 support in the following commit 5937131